src/components/VerificationCode/VerificationCode.css.js

```diff
@@ -0,0 +1,126 @@
+import styled from '../styled'
+import { getColor, rgba } from '../../styles/utilities/color'
+import Icon from '../Icon'
+import Tooltip from '../Tooltip'
+
+export const VerificationCodeFieldUI = styled('div')`
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: nowrap;
+  justify-content: center;
+  align-items: center;
+  position: relative;
+  width: 500px;
+  height: 74px;
+  border: 1px solid #c5ced6;
+  border-radius: 3px;
+  background-color: #fff;
+  box-shadow: 0 0 0 0 ${rgba(getColor('border'), 0)};
+  transition: box-shadow 100ms ease, border-color 100ms ease;
+
+  *,
+  *:before,
+  *:after {
+    box-sizing: inherit;
+  }
+
+  &:focus {
+    outline: none;
+  }
+
+  &:focus-within {
+    border-color: transparent;
+    box-shadow: 0 0 0 2px ${getColor('blue.500')};
+  }
+
+  &.not-valid {
+    border-color: transparent;
+    box-shadow: 0 0 0 2px #f23459;
```

Comment:
should the color be from our palette?

Based on colorway red.400 is the closest color: 
```
Finding the closest HSDS-React color for "f23459"
red.400: #f45b55
```

cc @jaredmcdaniel